_includes/_markdown/install-munit.md

```diff
@@ -4,12 +4,17 @@
 {% tab 'Scala CLI' %}
 You can require the entire toolkit in a single line:
 ```scala
-//> using dep "org.scala-lang::toolkit-test:0.1.7"
+//> using toolkit latest
 ```
+**However, MUnit won't be available in your source right away.**
```

Comment:
That message sounds scary, I think it would be better to only give a note that the dependency will be available in test files only, which are files within a `test` directory or that have the `.test.scala` extension.

Reply:
Something along the way:
```
MUnit being a testing framework, it is only available in test files: files in a `test` directory or that have the `.test.scala` extension. Refer to the [Scala CLI documentation]() to learn more about the test scope.
```

Reply:
Good point. I fixed it.